bci_tester/fips.py

```diff
@@ -86,6 +86,36 @@
     NONFIPS_GNUTLS_DIGESTS + FIPS_GNUTLS_DIGESTS
 )
 
+#: gcrypt digests that are not FIPS compliant
+NONFIPS_GCRYPT_DIGESTS: Tuple[str, ...] = (
+    "sm3",
+    "ripemd160",
+    "tiger",
+    "tiger2",
+    "tiger192",
+    "md4",
+    "whirlpool",
+    "gostr3411_94",
+    "stribog256",
+    "stribog512",
+    "md5",
+)
```

Comment:
Or you can add it here (imho simpler):
```suggestion
NONFIPS_GCRYPT_DIGESTS: Tuple[str, ...] = (
    "ripemd160",
    "tiger",
    "tiger2",
    "tiger192",
    "md4",
    "whirlpool",
    "gostr3411_94",
    "stribog256",
    "stribog512",
    "md5",
)

if OS_VERSION != "15.3":
    NONFIPS_GCRYPT_DIGESTS += ("sm3",)
```